package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "axios-har-tracker",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
Usually the initial package version is 0.1.0 ;)

Reply:
:)